frontend/scripts/router.js

```diff
@@ -65,35 +70,45 @@ const routes = {
     thunk: dispatchThunks(getDataPortalContext)
   },
   about: {
-    path: '/about',
-    page: 'about',
-    extension: 'jsx'
+    path: '/about/:section?',
+    page: 'static-content',
+    extension: 'jsx',
+    thunk: dispatchThunks(getPageStaticContent)
   },
   termsOfUse: {
     path: '/terms-of-use',
-    page: 'terms-of-use',
-    extension: 'jsx'
+    page: 'static-content',
+    extension: 'jsx',
+    thunk: dispatchThunks(getPageStaticContent)
   },
   dataMethods: {
     path: '/data-methods',
-    page: 'data-methods',
-    extension: 'jsx'
+    page: 'static-content',
+    extension: 'jsx',
+    thunk: dispatchThunks(getPageStaticContent)
   },
   faq: {
     path: '/FAQ',
-    page: 'FAQ',
-    extension: 'jsx'
+    page: 'static-content',
+    extension: 'jsx',
+    thunk: dispatchThunks(getPageStaticContent)
   },
   [NOT_FOUND]: {
-    path: '/404'
+    path: '/404',
```

Comment:
Please create a separate issue for us to test if this is actually returning an 404 http code